app/scripts/lib/notification-manager.js

```diff
@@ -72,7 +82,10 @@ export default class NotificationManager extends EventEmitter {
   _onWindowClosed(windowId) {
     if (windowId === this._popupId) {
       this._popupId = undefined;
-      this.emit(NOTIFICATION_MANAGER_EVENTS.POPUP_CLOSED);
+      this._popupAutomaticallyClosed = undefined;
```

Comment:
Why not `false`?

Reply:
Also, doesn't this make it so that `automaticallyClosed` is always `undefined`?

Reply:
Oops! Yes, good catch.

`false` works as well I suppose. I left it unassigned by default because when this isn't set, we don't know whether the next "window close" event is automatic or not. The options are "unknown" or "closed automatically", so `undefined` felt more appropriate

Reply:
Fixed!